test/fuzz/fuzz_runner.cc

```diff
@@ -87,7 +87,7 @@ void runCleanupHooks() {
 extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv) {
   // Before parsing gmock flags, set the default value of flag --gmock_verbose to "error".
   // This suppresses logs from NiceMock objects, which can be noisy and provide little value.
-  testing::GMOCK_FLAG(verbose) = "error";
+  GMOCK_FLAG_GET(verbose) = "error";
```

Comment:
Should probably be: `GMOCK_FLAG_SET(verbose, "error");`

Reply:
Done.